clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -49,6 +51,9 @@ class DockerRegistryClient {
   @Value('${dockerRegistry.client.timeout:60000}')
   int clientTimeout
 
+  @Value('${dockerRegistry.client.paginateSize:100}')
+  int paginateSize
```

Comment:
DockerRegistryClient has no @Component-based annotations, so unless it is being explicitly configured somewhere via @bean, it won't get picked up by component scanning.